### PR TITLE
Fix build for IDEA users: some dependencies had broken paths

### PR DESCRIPTION
**Pull request by alb-i986**
Compilation for the module client (Java) was failing on IDEA.

[X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: .idea/libraries/jetty.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <component name="libraryTable">
   <library name="jetty">
     <CLASSES>
-      <root url="jar://$PROJECT_DIR$/build/third_party/java/jetty/jetty-repacked.jar!/" />
+      <root url="jar://$PROJECT_DIR$/third_party/java/jetty/jetty-repacked.jar!/" />
```
Comment (lukeis) [inline .idea/libraries/jetty.xml]: this file also doesn't exist until you build it... the build folder one should exist for you, not sure why it doesn't

Comment (alb-i986) [inline .idea/libraries/jetty.xml]: > this file also doesn't exist until you build it...

Do you mean `third_party/java/jetty/jetty-repacked.jar`? It does exist actually:

```
$ git co master && ./go clean && find . -name jetty-repacked.jar
(in /Users/alb/workspace/selenium)
./third_party/java/jetty/jetty-repacked.jar
```

Comment (lukeis) [inline .idea/libraries/jetty.xml]: go clean doesn't clean everything. 

git clean -xdf

Comment (alb-i986) [inline .idea/libraries/jetty.xml]: I see, but I still think that the path should point to `third_party/` and not to `build/third_party/`

```
$ git clean -xdf && ./go clean && find . -name jetty-repacked.jar
-- no result whatsoever
$ ./go clean test_firefox && find . -name jetty-repacked.jar
-- still no result
$ ./go repack-jetty
$ find . -name jetty-repacked.jar
./build/third_party/java/jetty/jetty-repacked.jar
./third_party/java/jetty/jetty-repacked.jar
-- there it is!
$ ./go clean
$ find . -name jetty-repacked.jar
./third_party/java/jetty/jetty-repacked.jar
-- gone again!
```

Looks like you get `build/third_party/java/jetty/jetty-repacked.jar` right after executing `go repack-jetty`; but as soon as you run `go clean` it is lost, and all is left is `third_party/java/jetty/jetty-repacked.jar`.

Comment (lukeis) [inline .idea/libraries/jetty.xml]: You shouldn't be running clean so much ;) with the switch to buck it actually doesn't make sense to do it

Comment (lukeis) [inline .idea/libraries/jetty.xml]: I think I may even remove the 'clean' target :)

Comment (alb-i986) [inline .idea/libraries/jetty.xml]: Uhm ok then :)
I've reverted the changes for jetty.
Please check it out.

**Pull request merged**